docker/rust-toolchain

```diff
@@ -1,2 +0,0 @@
-[toolchain]
```

Comment:
I think this needs to get moved to the root
THere is a symlink from the root rust-toolchain to this

Reply:
That explains the build failure :)